run-tests.sh

```diff
@@ -116,7 +116,7 @@ test_scanner() {
 
 launch_sonarqube() {
   info "Starting SonarQube in container $sonarqube_container_name in detached mode..."
-  docker run --network="$network" --name="$sonarqube_container_name" -d -p "$port:9000" sonarqube
+  docker run --network="$network" --name="$sonarqube_container_name" -d -p "$port:9000" sonarqube:enterprise
```

Comment:
Did the change to run the enterprise edition stem from the tests we use from `sonar-scanning-examples`?

Reply:
We could use any SQ version in the tests but I think it just makes sense to use SQ Enterprise Edition because more languages/plugins are covered this way (so more potential issues will be exposed by the test